src/lib/with-layout.js

```diff
@@ -84,14 +84,18 @@ export default function withLayout (Component) {
       this.setState({ toggleShowClass: !this.state.toggleShowClass })
     }
 
-    toggleMiniBag() {
-      this.setState({ minibagDisplayed: !this.state.minibagDisplayed })
+    toggleMiniBag(displayed) {
+      this.props.dispatch(toggleMiniBag(displayed))
     }
 
     deleteItem(event) {
```

Comment:
Tiny change and not code you've touched but can we get a space before the argument list just for consistency